lib/minify.js

```diff
@@ -1,10 +1,24 @@
 "use strict";
 
 var to_ascii = typeof atob == "undefined" ? function(b64) {
-    return new Buffer(b64, "base64").toString();
+    if (Buffer.from && Buffer.from !== Uint8Array.from) {
+      // Node >= 4.5.0
+      return Buffer.from(b64, "base64").toString("ascii");
```

Comment:
although the function is called `to_ascii`, converting the buffer to ascii instead of utf8 breaks parsing of source maps with valid unicode characters in them. i think this branch should just do `.toString()` just like the one below.

Reply:
Browser-side `atob()` and `btoa()` methods do not support non-ascii.

Refs:
* https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/atob
* https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/btoa

So parsing of source maps with valid unicode characters is already broken when this method is executed in a browser environment, and changing this single `.toString()` won't help much. I will update it, though, to keep the behavior closer to the original one.

Reply:
Done.